html/controls/operator.js

```diff
@@ -1064,28 +1070,21 @@ function createScoreBoardViewPreviewRows(table, type) {
 			{ text: "8%", value: "8" },
 			{ text: "10%", value: "10" }
 		]}});
-	var backgroundStyle = $sb("ScoreBoard.Settings.Setting(ScoreBoard." + type + "_BackgroundStyle)").$sbControl("<label>Background Style: </label><select>", { sbelement: {
```

Comment:
The background is still showing as black to white.

Reply:
Where? I have solid black on both legacy and WS view.

Reply:
Weird, it's showing as all black now - even though I hard refreshed several times before.